internal/services/mssql/mssql_server_resource_test.go

```diff
@@ -762,6 +784,72 @@ resource "azurerm_mssql_server" "test" {
 `, data.RandomInteger, data.Locations.Primary)
 }
 
+func (MsSqlServerResource) password(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+provider "azurerm" {
+  features {}
+}
+
+provider "azuread" {}
+
+data "azurerm_client_config" "test" {}
+
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-mssql-%[1]d"
+  location = "%[2]s"
+}
+
+resource "azurerm_mssql_server" "test" {
+  name                = "acctestsqlserver%[1]d"
+  resource_group_name = azurerm_resource_group.test.name
+  location            = azurerm_resource_group.test.location
+  version             = "12.0"
+
+  administrator_login          = "missadministrator"
+  administrator_login_password = "thisIsKat11"
```

Comment:
I'm not sure if having `azuread_authentication_only = true` and `administrator_login_password` present is a feasible state. My bug report's initial state is aad auth is true and there is no additional administrator_login setting.

Reply:
I assume that it is possible because in the [Terraform documentation](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/mssql_server#azuread_authentication_only) it is stated that  `When true, the administrator_login and administrator_login_password properties can be omitted.` . I assuming this means `omitted` means azure authentication only, not `omitted` means both. Also, the test case `TestAccMsSqlServer_passwordUpdate` passed.